back/app/services/permissions_service.rb

```
@@ -83,7 +83,7 @@ def remove_extras_actions(scope, actions = nil)
   def add_missing_actions(scope, actions = nil)
     missing_actions = missing_actions(scope, actions)
     permissions_hashes = missing_actions.map { |action| { action: action } }
-    Permission.create(permissions_hashes) { |p| p.permission_scope = scope }
+    Permission.create!(permissions_hashes) { |p| p.permission_scope = scope }
```

I think the answer is no, but just to make sure: This wouldn't fail in expected cases? I mean, we're not counting on this to fail silently for normal operation?

---

For missing actions, it should be possible to create the permissions. I doubt `create` was used instead of `create!` on purpose. I can't be 100% sure this will always succeed, but I would propose to monitor issues well after releasing (this would pop up in Sentry). If there would be such cases, I would propose to make this more explicit with an if-statement.

---

Alright!